dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -304,6 +305,31 @@ def __init__(self, label_mapping=None, parameters=None):
         :return: None
         """
 
+        # https://github.com/tensorflow/tensorflow/issues/34697
```

Comment:
did you try another method down below on the same url? It doesn't require saving a file, and we can remove tempfile

Reply:
I think this is appropriate as we may be transferring the model between processes which don't' share a memory space.